src/Symfony/Component/Validator/Resources/translations/validators.sv.xlf

```diff
@@ -302,6 +302,14 @@
                 <source>An empty file is not allowed.</source>
                 <target>En tom fil är inte tillåten.</target>
             </trans-unit>
+            <trans-unit id="79">
+                <source>The host could not be resolved.</source>
+                <target>Domänen kunde inte hittas.</target>
```

Comment:
Not at all @javiereguiluz. In the contrary. I'm impressed that you took your time to review this Swedish translation. (It's quite rare that a non-Swedish speaker complains about my Swedish :smile: )

I've updated to "Värddatorn". Thank you all for giving me quick input in this. Beers is on me next time we meet.